server/src/main/java/io/crate/metadata/table/Operation.java

```diff
@@ -57,18 +59,22 @@ public enum Operation {
     public static final EnumSet<Operation> SYS_READ_ONLY = EnumSet.of(READ);
     public static final EnumSet<Operation> READ_ONLY = EnumSet.of(READ, ALTER_BLOCKS);
     public static final EnumSet<Operation> CLOSED_OPERATIONS = EnumSet.of(ALTER_OPEN, ALTER_CLOSE, ALTER_TABLE_RENAME);
-    public static final EnumSet<Operation> BLOB_OPERATIONS = EnumSet.of(READ, OPTIMIZE, ALTER, ALTER_REROUTE, DROP);
+    public static final EnumSet<Operation> BLOB_OPERATIONS = EnumSet.of(READ, OPTIMIZE, ALTER, ALTER_REFRESH_INTERVAL,
+        ALTER_NUMBER_OF_REPLICAS, ALTER_REROUTE, DROP);
     public static final EnumSet<Operation> READ_DISABLED_OPERATIONS = EnumSet.of(UPDATE, INSERT, DELETE, DROP, ALTER,
-        ALTER_OPEN, ALTER_CLOSE, ALTER_REROUTE, ALTER_BLOCKS, REFRESH, OPTIMIZE);
-    public static final EnumSet<Operation> WRITE_DISABLED_OPERATIONS = EnumSet.of(READ, ALTER, ALTER_OPEN, ALTER_CLOSE,
-        ALTER_BLOCKS, ALTER_REROUTE, SHOW_CREATE, REFRESH, OPTIMIZE, COPY_TO, CREATE_SNAPSHOT);
+        ALTER_REFRESH_INTERVAL, ALTER_NUMBER_OF_REPLICAS, ALTER_OPEN, ALTER_CLOSE, ALTER_REROUTE,
+        ALTER_BLOCKS, REFRESH, OPTIMIZE);
+    public static final EnumSet<Operation> WRITE_DISABLED_OPERATIONS = EnumSet.of(READ, ALTER, ALTER_REFRESH_INTERVAL,
+        ALTER_NUMBER_OF_REPLICAS, ALTER_OPEN, ALTER_CLOSE, ALTER_BLOCKS, ALTER_REROUTE, SHOW_CREATE, REFRESH,
+        OPTIMIZE, COPY_TO, CREATE_SNAPSHOT);
     public static final EnumSet<Operation> METADATA_DISABLED_OPERATIONS = EnumSet.of(READ, UPDATE, INSERT, DELETE,
         ALTER_BLOCKS, ALTER_OPEN, ALTER_CLOSE, ALTER_REROUTE, REFRESH, SHOW_CREATE, OPTIMIZE);
-    public static final EnumSet<Operation> SUBSCRIBED_IN_LOGICAL_REPLICATION = EnumSet.of(
-        READ, ALTER_BLOCKS, ALTER_REROUTE, OPTIMIZE, REFRESH, COPY_TO, SHOW_CREATE, ALTER_OPEN);
-    public static final EnumSet<Operation> PUBLISHED_IN_LOGICAL_REPLICATION = EnumSet.of(
-        READ, UPDATE, INSERT, DELETE, DROP, ALTER, ALTER_BLOCKS, ALTER_OPEN, ALTER_CLOSE, ALTER_REROUTE, REFRESH,
-        SHOW_CREATE, COPY_TO, OPTIMIZE, RESTORE_SNAPSHOT, CREATE_SNAPSHOT);
+    public static final EnumSet<Operation> SUBSCRIBED_IN_LOGICAL_REPLICATION = EnumSet.of(READ, ALTER_BLOCKS,
```

Comment:
not sure about `ALTER_OPEN` in both pub/sub operations - it was added to manually re-open table if `DROP SUB` closed it and failed mid way - then `DROP SUB` in 3 requests was partially [reverted](https://github.com/crate/crate/pull/12410) as we have new approach for it. Looks like I forgot to remove alter_open from local replication operations.

Reply:
yes `ALTER OPEN` is not needed anymore.